content/articles/setting-up-accounts-for-clients.markdown

```diff
@@ -0,0 +1,90 @@
+---
+title: Creating accounts for clients
+excerpt: How to create and manage multiple accounts for clients.
+categories:
+- Account
+---
+
+# Creating accounts for clients
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+<note>
+In order to have multiple members on a single account, you must first [be subscribed to an eligible plan](https://dnsimple.com/pricing) for that account.
+</note>
+
+Creating a separate account is a flexible and secure way for agencies and freelancers to share domain management with their clients.
+
+Keep in mind that when you add your client as a user to an account, the new member will have full access rights on every resource attached to the account, including domains, contacts, SSL certificates, etc.
+
+The member will also have full-admin access to the account, including the ability to access and change billing information and account details. The user will retain his login credentials and 2-factor authentication configuration.
+
+<warning>
+Each customer must have a separate account, otherwise if you add more than one customer to the same account each customer will be able to manage other customers' domains.
+</warning>
+
+
+## Creating a separate account for your client
+
+First, you must add another account to your user profile by taking the following steps.
+
+<div class="section-steps" markdown="1">
+1. Visit the settings for your user profile by clicking the <label>Settings</label> link in the dropdown menu.
+
+    ![Settings link](/files/access-user-settings.jpg)
+
+1. Once there, you follow the <label>Add new account</label> link.
+
+    ![Add New Account link](/files/add-new-account-link.jpg)
+
+1. At this point, you will be presented with a form where you should fill in with the details for your new account.
+
+    ![Add New Account form](/files/add-new-account-form.jpg)
+
+1. Once you've submitted the new account form, the account is created and you are free to activate it by clicking the <label>Select a plan</label>. You may also configure other parts of your account from the top navigation menu before activating your account.
+
+    ![Add New Account details](/files/add-new-account-details.jpg)
+
+</div>
+  
+## Inviting your client to DNSimple
+  
+  Verify with your client that is a __registered__ DNSimple.
+  
+  If needed, [point your client to the signup page](https://dnsimple.com/signup).
+  
+  <note>It is not necessary that your client have an active subscription, just that they have already signed up for the service.</note>
+  
+## Adding clients to the new account
+
+<div class="section-steps" markdown="1">
+1. Once you've subscribed the new account to an eligible plan, visit the account page by clicking the name of the account in the dropdown menu.
```

Comment:
@meeunier this is the standard we have throughout the support site right now. It's not too big of a burden, especially since many of these tasks don't necessarily need to be given step-by-step if the customer is already familiar with them.